### ISSUE TITLE
Problem regarding issue #1288, column not found

### ISSUE THREAD

**Issue by fballiano** [OpenMage/magento-lts#1562]
Just upgraded a project to 20.0.10, issue https://github.com/OpenMage/magento-lts/issues/1288 made the Mage_Setup module bump from 1.6.0.0 to 1.6.0.1 to run a setup_script.

On my installation the script was not run (cache were reset multiple times but nothing changed), the record in the core_resource table is still old:
<img width="588" alt="Schermata 2021-04-21 alle 14 28 53" src="https://user-images.githubusercontent.com/909743/115561948-f0ec6c00-a2ad-11eb-84c8-009a5378a79f.png">

and the checkout is broken:
![Schermata 2021-04-21 alle 15 09 47](https://user-images.githubusercontent.com/909743/115561828-d74b2480-a2ad-11eb-9fd8-a5ea2097ee89.jpg)

I've no idea why the setup script is not running.

**Comment by ADDISON74** [comment]
That commit was not merged at least in OpenMage 20.0.8 even it has a milestone for that version. I asked what happened here #1480 but I did not get any replies.

Normally when you delete the cache and if there is a new version in config.xml file Magento will run the installation script. I did not check if all the modified files are in 20.0.10 but for sure something happened with that commit.

**Comment by fballiano** [comment]
if was released in 20.0.8 (I had the problem in 20.0.8 but then upgraded to 20.0.10 just to check if maybe it was solved).

the config.xml of mage_checkout module is correct (1.6.0.1) but the setup_script is not run and the checkout is broken.

**Comment by ADDISON74** [comment]
I checked the archives for 20.0.7 and 20.0.8. The script installer was introduced in 20.0.8. I remember I installed this version just after releasing and I reported a few issues. I did not find the changes in Backend related to #1480 and this is the reason I asked in the commit page. Looking over the code it looks just fine.

If you don't alter manually the table to add the new column that website won't work in checkout process. I will install this new version from scratch and see what is happening. For me one thing is clear some changes require more testing before releasing. In the last month, several high issues have been identified in the code and it has been necessary to launch new variants just to fix the errors.

**Comment by fballiano** [comment]
yep, I can add the column and bump up the record in the core_resource table but still I think it's important to report the (possible) bug.

but I really can't see any reason why it shouldn't run the setup_script :-\

**Comment by ADDISON74** [comment]
I only once encountered a similar situation with an extension from Aheadworks. I deleted the record of the extension from the core_resources table to play a little and I was very surprised to find that the versions were not modified and the installation script was not running. I took it close to the numbers by modifying them directly in the table based on the file names. In the end I found the variant from which the last version could be installed. I wrote it on a paper just in case the table is ever deleted to know how to reinstall that extension without deleting the table and taking other steps to modify the database.
